config.json

```diff
@@ -6,7 +6,7 @@
     {
       "slug": "hello-world",
       "uuid": "f2936612-edc4-44f9-a403-c60f7c6486e4",
-      "core": true,
+      "core": false,
```

Comment:
Oh, I need to fix that.  I had to do a lot of manual merge conflict fixes while rebasing this.

Reply:
Fixed